src/test/java/unicash/model/category/UniqueCategoryListTest.java

```diff
@@ -196,7 +196,16 @@ public void toStringMethod() {
     @Test
     public void getSize_test() {
         UniqueCategoryList categoryList = new UniqueCategoryList();
-        assertEquals(uniqueCategoryList.getSize(), 0);
+        assertEquals(categoryList.getSize(), 0);
+    }
+
+    @Test
+    public void joinCategoriesAsStringTest() {
+        UniqueCategoryList categoryList = new UniqueCategoryList();
+        categoryList.add(ENTERTAINMENT);
+        categoryList.add(EDUCATION);
+        assertEquals(categoryList.joinCategoriesAsString(),
+                ENTERTAINMENT + "," + EDUCATION);
```

Comment:
Believe that `assertEquals` takes in expected, then outcome, so it should be the second argument first